core/api/src/services/notifications/index.ts

```diff
@@ -194,46 +199,38 @@ export const NotificationsService = (): INotificationsService => {
     transaction,
   }: NotificatioSendTransactionArgs): Promise<true | NotificationsServiceError> => {
     try {
-      const settings = await getUserNotificationSettings(recipient.userId)
-      if (settings instanceof Error) {
-        return settings
-      }
-
-      const hasDeviceTokens = settings.pushDeviceTokens.length > 0
-      if (!hasDeviceTokens) return true
-
       const type = getPushNotificationEventType(transaction)
       if (!type) return true
 
+      const settlementAmount = new ProtoMoney()
+      settlementAmount.setMinorUnits(transaction.settlementAmount)
+      settlementAmount.setCurrencyCode(transaction.settlementCurrency)
+
       const displayAmountMajor = transaction.settlementDisplayAmount
       const displayCurrency = transaction.settlementDisplayPrice.displayCurrency
-      const displayAmountMinor = roundToBigInt(
+      const displayAmountMinor = Math.round(
         majorToMinorUnit({ amount: Number(displayAmountMajor), displayCurrency }),
       )
+      const displayAmount = new ProtoMoney()
+      displayAmount.setMinorUnits(displayAmountMinor)
+      displayAmount.setCurrencyCode(displayCurrency)
 
-      const { title, body } = createPushNotificationContent({
-        type,
-        userLanguage: settings.language,
-        amount: {
-          amount: roundToBigInt(transaction.settlementAmount),
-          currency: transaction.settlementCurrency,
-        },
-        displayAmount: {
-          amountInMinor: displayAmountMinor,
-          displayInMajor: displayAmountMajor,
-          currency: displayCurrency,
-        },
-      })
+      const tx = new TransactionInfo()
+      tx.setUserId(recipient.userId)
+      tx.setType(type)
+      tx.setSettlementAmount(settlementAmount)
+      tx.setDisplayAmount(displayAmount)
 
-      const result = await pushNotification.sendFilteredNotification({
```

Comment:
Removing this line causes https://github.com/GaloyMoney/galoy/blob/1852210584ae7583fbe0f65c9ecb16b3ff9a25ba/core/api/test/integration/app/wallets/send-lightning.spec.ts#L899 to fail...

cc @vindard